src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -6104,7 +6114,7 @@ static inline INT32 GetNextSleepTime(SystemTimer *timer, ThreadIdleScheduler *sc
     //
     // [9],{2},[8],{1},[9],{1},[9],{3},[7],{2},[8],{3},[7],{2},[8],{2},[8],{3},[7],{2},[8],...
     //
-    INT32 procTime = (INT32)(SystemTimer_GetTime(timer) - startTime);
```

Comment:
Please change of the type of `procTime` back to`INT32`.

Reply:
done